CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 * Added `NavigationMapView.recenterMap()`. A helpful function for recenter the camera if it becomes uncentered.
 * Moved `RouteController.tunnelSimulationEnabled` to `RouteController.tunnelIntersectionManager.tunnelSimulationEnabled`. [#1489](https://github.com/mapbox/mapbox-navigation-ios/pull/1489/)
 * Deprecated `NavigationViewController.usesNightStyleInsideTunnels`. Style switching in now enabled as long as `RouteController.tunnelIntersectionManager.tunnelSimulationEnabled` is true.
+* Increases the minimum deployment target to iOS 9. [#1494](https://github.com/mapbox/mapbox-navigation-ios/pull/1494)
```

Comment:
35c066a765e771fd063c89983b0d59c955b28447 clarifies that this only affects Core Navigation.